app/src/main/java/com/fstyle/androidtrainning/screen/tabhome/TabHomePresenter.java

```diff
@@ -20,4 +35,81 @@ public void onStart() {
     @Override
     public void onStop() {
     }
+
+    @Override
+    public void setMoviesApi(MoviesApi moviesApi) {
+        mMoviesApi = moviesApi;
+    }
+
+    @Override
+    public void getListNowPlaingMovie() {
+        mMoviesApi.getListMovies(Constant.CATEGORY_NOW_PLAYING, Constant.API_KEY, Constant.LANGUAGE,
+                Constant.DEFAULT_PAGE).enqueue(new Callback<GetListMoviesResponse>() {
+            @Override
+            public void onResponse(Call<GetListMoviesResponse> call,
+                    Response<GetListMoviesResponse> response) {
+                List<Movie> listNowPlayingMovie = response.body().getResults();
+                mHomeView.onListNowPlayingMovieSuccess(listNowPlayingMovie);
+            }
+
+            @Override
+            public void onFailure(Call<GetListMoviesResponse> call, Throwable t) {
+                Log.e(TAG, "onFailure: ", t);
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "onFailure: " appears 4 times in this file; the first occurrence is on line 57
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals